oauth2client/contrib/gce.py

```diff
@@ -85,32 +142,126 @@ class AppAssertionCredentials(AssertionCredentials):
     information to generate and refresh its own access tokens.
     """
 
-    @util.positional(2)
-    def __init__(self, scope='', **kwargs):
+    def __init__(self,
+                 scope=None,
+                 service_account_email='default',
+                 service_account_info=None,
+                 **unused_kwargs):
         """Constructor for AppAssertionCredentials
 
         Args:
-            scope: string or iterable of strings, scope(s) of the credentials
-                   being requested. Using this argument will have no effect on
-                   the actual scopes for tokens requested. These scopes are
-                   set at VM instance creation time and won't change.
+            scope:
+                string or iterable of strings, scope(s) of the credentials
```

Comment:
Can we avoid this? One of Python's greatest mistakes is the conflation of strings and iterables of strings; is there any reason we should perpetuate it here?